extension/address/deleteAddresses.js

```diff
@@ -0,0 +1,38 @@
+const InternalError = require('./../common/Error/InternalError')
+const NotFoundError = require('./../common/Error/NotFoundError')
+
+/**
+ * @param {SDKContext} context
+ * @param {{ids: number[]}} input
+ * @return {Promise<>}
+ */
+module.exports = async (context, { ids }) => {
+  // Read all addresses from user storage
```

Comment:
Please sanitize `ids` to be Array of objects

Reply:
done